### PR TITLE
feat: implement all core integrations

### PR DESCRIPTION
**Pull request by Pagebakers**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,7 +88,8 @@
       "@tanstack/query-core": "^4.29.1",
       "zod": "^3.21.4",
       "json-machete": "0.94.5",
-      "@graphql-mesh/store": "0.94.5"
+      "@graphql-mesh/store": "0.94.5",
+      "react": "18.2.0"
```
Comment (fiam) [inline package.json:92]: Is this intentional?

Comment (Pagebakers) [reply package.json:92]: Yes, it's actually added to resolutions, but not sure if latest pnpm respects this. This makes sure all projects use the same react version. I had issues when upgrading nextjs.

File: testapps/nextjs-integrations/public/wundergraph.svg
Comment (Aenimus) [inline testapps/nextjs-integrations/public/wundergraph.svg:1]: @Pagebakers old logo

**Review by fiam — APPROVED** [review]
LGTM, this new API looks so much nicer

File: testapps/nextjs-integrations/public/wundergraph.svg
Comment (Pagebakers) [reply testapps/nextjs-integrations/public/wundergraph.svg:1]: Out of scope for this PR, most examples actually have the old logo still I believe

**Review by Aenimus — APPROVED** [review]

**Pull request merged**